<launch>
  <rosparam command="load" file="$(find rpi_led_consensus)/config/config.yaml" param="consensus_params"/>
</launch>
